druid-shell/Cargo.toml

```diff
@@ -43,8 +46,9 @@ cocoa = "0.20.0"
 objc = "0.2.5"
 cairo-rs = { version = "0.8.0", default_features = false }
 
+# TODO(x11/dependencies): only use feature "xcb" if using XCB
 [target.'cfg(target_os="linux")'.dependencies]
-cairo-rs = {  version = "0.8.0", default_features = false }
+cairo-rs = {  version = "0.8.0", default_features = false, features = ["xcb"] }
```

Comment:
it's annoying that we need to always have these on for linux; it means that our x11 backend depends on gtk. I'm not sure what the best solution is.

Reply:
Hmm, I guess we could technically just split that and make more-specific `cfg` targets?

Reply:
unfortunately you can't use feature targets in this way. :/

Reply:
Ah, dang. Sounds like it should be a feature. Wonder if there's a Rust / Cargo feature request laying around somewhere...